Makefile.am

```diff
@@ -93,6 +94,21 @@ mcs-do-compiler-tests:
 bootstrap-world: compiler-tests
 	$(MAKE) install
 
+install:
+	echo $(SUBDIRS)
+	for mydir in $(SUBDIRS) ; do \
```

Comment:
I feel like this whole block could be simplified to:

```bash
for mydir in $(filter-out $(noinst_SUBDIRS),$(SUBDIRS)); do \
  (cd $${mydir} && ${MAKE} install) \
done
```

Reply:
The simplification still works, so committed.